Mathlib/MeasureTheory/Constructions/EventuallyMeasurable.lean

```diff
@@ -0,0 +1,117 @@
+/-
+Copyright (c) 2024 Felix Weilacher. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Felix Weilacher
+-/
+
+import Mathlib.MeasureTheory.MeasurableSpace.Defs
+import Mathlib.Order.Filter.CountableInter
+
+/-!
+# Measurability modulo a filter
+
+In this file we consider the general notion of measurability modulo a σ-filter.
+Two important instances of this construction are null-measurability with respect to a measure,
+where the filter is the collection of co-null sets, and
+Baire-measurability with respect to a topology,
+where the filter is the collection of comeager (residual) sets.
```

Comment:
I think I am confused. For me, Baire measurable functions are functions which are measurable for the Baire sigma-algebra, i.e., the smallest one that makes compactly supported continuous functions into measurable functions. I don't see the relationship with the notion you are introducing in this file. Maybe you are thinking of a different notion of Baire-measurability?

Reply:
Yeah, this is an unfortunate situation. I have more commonly seen what you are talking about called "Baire sets/functions". The notion I mention for sets is sometimes called the "Baire property" but for functions it is usually called "Baire measurable". 

You could definitely make the case for using "Baire measurable" to refer to either notion. Googling the phrase shows a somewhat even number of occurrences for both. 

I am not sure what the right names for mathlib are, but it shouldn't affect this PR other than maybe requiring a change to this docstring.

Reply:
Sure, I am just asking for a clarification in the docstring of which notion of Baire-measurability you are talking about here.

Reply:
ok, i added a line about this.